src/test/java/org/simulator/sbml/SBMLTestSuiteTest.java

```diff
@@ -37,30 +39,33 @@
     public static final String RELATIVE = "relative";
     private static final Logger logger = LoggerFactory.getLogger(TestUtils.class);
     private static final String SBML_TEST_SUITE_PATH = "SBML_TEST_SUITE_PATH";
-    private static final double THRESHOLD = 0.001;
+    private static final double TOLERANCE_FACTOR = 1E-3;
 
     @Before
-    public void setUp(){ }
+    public void setUp() {
+    }
 
     @After
-    public void tearDown(){ }
+    public void tearDown() {
+    }
```

Comment:
Do we need to override these methods if we do nothing? As far as I can remember, JUnitTestRunner does nothing if there is no @Before, @After, @SetUp, or @TearDown methods are specified.